### DISCUSSION TITLE
Russian translation of R data.table docs?

### DISCUSSION THREAD

**Discussion by tdhock** [liquidcarbon/liquidcarbon#16] [General]
Hi @liquidcarbon I am leading a NSF POSE project about expanding the open-source ecosystem around R data.table, which is a free/open-source software package for efficient data reading/manipulation/analysis/etc. As a part of this project, we would like to translate the messages/vignettes/slides/etc to languages other than English. Here is the call for proposals https://rdatatable-community.github.io/The-Raft/posts/2023-10-17-translation_announcement-toby_hocking/ So far there are projects for Chinese, French, and Portuguese (Brazil).
I would like to hire some people to work on a Russian translation. I heard from Igor Rupniewski that you may be capable. The pay is $500 per participant. Would you be interested to lead or participate? Do you know anyone else who may be capable (R programming + Russian language skills) and interested? Thanks in advance!

**Comment by liquidcarbon** [comment]
Hey, thanks for reaching out, cool initiative!  I'm checking with a few friends.
